### PR TITLE
feat: Improving fio_zone_throughput_avg_lat benchmark.

### PR DESCRIPTION
**Pull request by MaisenbacherD**
Refined max_open_zone and queue_depth parameters.
Generating throughput and latency plots for all parameter combinations.
Added plots for latency percentiles.

Signed-off-by: Dennis Maisenbacher <dennis.maisenbacher@wdc.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,9 +125,9 @@ fio_zone_throughput_avg_lat
     the throughput and latency in the csv report:
       - Read/write
       - Seq/Random
-      - BS: 4K, 8K, 16K, 64K, 128K
-      - max_open_zone: 1, 2, 4, 8, 12 (only for writes)
-      - QD: 1, 2, 4, 8 (skipping QD's > max_open_zones)
+      - BS: 4K, 8K, 14K, 64K, 128K
```
Comment (metaspace) [inline README.md:128]: I think this comment update is wrong. Should not be BS?

**Review by metaspace — CHANGES_REQUESTED** [review]
Looks good, but for a doc update that when to the wrong line :)

File: README.md (hunk 1 above)
Comment (MaisenbacherD) [reply README.md:128]: You are right. QD has to be updated.

**Comment by MaisenbacherD** [comment]
After an offline discussion with @metaspace we agreed that `QD: 1, 2, 4, 8, 14, 16, 32, 64` makes more sense. QD 14 is useful because of writes dropping QD > max_open_zones.

**Pull request merged**